python/composio/utils/warnings.py

```diff
@@ -24,17 +24,14 @@ def _fetch_latest_version():
 
 
 def create_latest_version_warning_hook(version: str):
+    if os.environ.get("COMPOSIO_DISABLE_VERSION_CHECK", "false").lower() != "false":
```

Comment:
The condition has been changed from explicitly checking for `\"true\"` to checking for `!= \"false\"`. This means any value other than \"false\" will now disable version checking. Is this change in behavior intentional? If so, it should be documented as it could affect existing users who might be using other values.